### PR TITLE
Allow admin to delete activation/reset keys and generate reset key

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb/issues/3349

### generate reset key
Screen Shot 2022-12-29 at 3 23 09 PM

### info/delete activation/reset keys
Screen Shot 2022-12-29 at 3 23 24 PM

### CODE REVIEW

File: src/main/webapp/app/pages/userPage/UserPage.tsx [hunk 1]
```diff
@@ -340,73 +425,7 @@ export default class UserPage extends React.Component<IUserPage> {
         country: values.country,
       };
       this.getUserStatus = PromiseStatus.pending;
-      await this.verifyUserEmail();
```
Comment (calvinlu3) [inline src/main/webapp/app/pages/userPage/UserPage.tsx:343]: We can remove `async` from function now that this call is removed

File: src/main/webapp/app/pages/userPage/UserPage.tsx [hunk 2]
```diff
@@ -470,19 +489,45 @@ export default class UserPage extends React.Component<IUserPage> {
     if (this.trialInitiated) {
       this.showTrialAccountModal = true;
     } else {
-      this.showTrialAccountConfirmModal = true;
+      this.simpleConfirmModalType = SimpleConfirmModalType.INITIATE_TRIAL;
+    }
+  }
+
+  @autobind
+  onConfirmSimpleConfirmModel() {
```
Comment (calvinlu3) [inline src/main/webapp/app/pages/userPage/UserPage.tsx:497]: Typo
```suggestion
  onConfirmSimpleConfirmModal() {
```

**Review by calvinlu3 — APPROVED** [review]

File: src/main/webapp/app/pages/userPage/UserPage.tsx (hunk 1 above)
Comment (calvinlu3) [reply src/main/webapp/app/pages/userPage/UserPage.tsx:343]: Nevermind this, was fixed in recent commit

Comment (zhx828) [reply src/main/webapp/app/pages/userPage/UserPage.tsx:343]: Yup, just fixed actually. Thanks for catching this.

**Pull request merged**